src/cmd/sanity_test/sanity_tests.go

```diff
@@ -46,13 +46,19 @@ func main() {
 	}
 }
 
-func checkEmailRestoration(client *msgraphsdk.GraphServiceClient, testUser, folderName string, startTime time.Time) {
+func checkEmailRestoration(
+	client *msgraphsdk.GraphServiceClient,
+	testUser,
+	folderName string,
+	startTime time.Time,
+) {
 	var (
 		messageCount  = make(map[string]int32)
 		restoreFolder models.MailFolderable
 	)
 
-	builder := client.UsersById(testUser).MailFolders()
+	user := client.UsersById(testUser)
+	builder := user.MailFolders()
```

Comment:
We should avoid builders and construct the call where needed.  There's not a lot of re-use going on here to benefit from.

Reply:
its a paginated call. on line - https://github.com/alcionai/corso/blob/7207be77155b90911596847c7b77bce5f0714610/src/cmd/sanity_test/sanity_tests.go#L101
new builder value is inserted. So initialising the builder before the `for` loop was required.